spec/integration_spec.rb

```diff
@@ -55,4 +55,29 @@
     it { expect(response[:result]).to include '<div>' }
     it { expect(response[:expectation_results]).to include({binding: 'PonerUnaDeCada', inspection: 'HasUsage', result: :passed}) }
   end
+
+  context 'when the program is Empty' do
```

Comment:
I'd change it to `when submission is empty and no extra code is given`, I think it's a better description of the scenario